src/Session/session.py

```diff
@@ -0,0 +1,49 @@
+from os import listdir
+from src.Constants.commands_messages import \
+    SUGGESTED_ACTIONS_MESSAGE
+from src.Constants.constants import \
+    CREATE_PROFILE, \
+    EMPTY_STRING, \
+    EXIT, \
+    JSON_EXTENSION, \
+    PROFILES_PATH, \
+    WITHIN_QUOTES
+from src.Constants.session_messages import *
+from src.Utils.utils import listSepByTab
+
+class Session:
+    def __init__(self):
+        self._profiles = self._getSavedProfiles()
+
+    ''' Fetches currently saved profiles and returns their names in an array. '''
+    def _getSavedProfiles(self):
+        return [file.replace(JSON_EXTENSION, EMPTY_STRING) for file in sorted(listdir(PROFILES_PATH))]
```

Comment:
So when the user creates and saves a profile, it'll get stored in `_getSavedProfiles()`? And they'll be able to access those profiles through here? Is it saved as like a JSON (just wondering since you're using `.json`)?

Reply:
I see, thank you for breaking down the steps for me! :)